src/pages/sangminLee/Main/Main.scss

```
@@ -0,0 +1,360 @@
+            }
+          }
+          .side-story:nth-child(5) {
+            .story-profile {
+              background-image: url('../../../../public/images/sangminLee/ggomi4.jpeg');
+            }
+          }
+        }
+      }
+      .side-recommandsbox {
+        margin-top: 15px;
+        height: 300px;
+        width: 300px;
+        border: 3px solid #ddd;
+        background-color: white;
+        .side-recommandstitle {
+          background-color: white;
+          padding: 0 10px;
+          height: 40px;
+          display: flex;
+          justify-content: space-between;
+          align-items: center;
+        }
+        .side-recommandslist {
+          list-style-type: none;
+          padding-inline-start: 0;
+          padding-left: 10px;
+          overflow: auto;
+          margin-block-end: 0;
+          margin-block-start: 0;
+          height: 255px;
+          .side-recommand:nth-child(1) {
+            .recommand-profile {
+              background-image: url('../../../../public/images/sangminLee/ggomi5.jpeg');
+            }
+          }
+          .side-recommand:nth-child(2) {
+            .recommand-profile {
+              background-image: url('../../../../public/images/sangminLee/ggomi6.jpeg');
+            }
+          }
+          .side-recommand:nth-child(3) {
+            .recommand-profile {
+              background-image: url('../../../../public/images/sangminLee/ggomi7.jpeg');
+            }
+          }
+          .side-recommand:nth-child(4) {
+            .recommand-profile {
+              background-image: url('../../../../public/images/sangminLee/ggomi8.jpeg');
+            }
+          }
+          .side-recommand:nth-child(5) {
+            .recommand-profile {
+              background-image: url('../../../../public/images/sangminLee/ggomi9.jpeg');
+            }
+          }
+        }
+        .side-information {
+          margin: 15px 0;
+          opacity: 0.4;
+          width: 300px;
+        }
+      }
+      .side-company {
+        opacity: 0.4;
+        width: 300px;
+      }
+    }
+  }
+}
+
+.blur {
+  opacity: 0.6;
+}
+
+.title-name {
+  font-size: 18px;
+}
+
+.title-autor {
+  font-size: 14px;
+  margin-left: 2px;
+  margin-top: 2px;
+}
+
+.side-story {
+  display: flex;
+}
+
+.story-profile {
+  margin-bottom: 15px;
+  width: 55px;
+  height: 55px;
+  border: 1px solid #fafafa;
+  border-radius: 100%;
+  background-size: 55px;
+}
+
+.story-title {
+  margin-left: 10px;
+}
+
+.story-name {
+  margin-top: 10px;
+  font-size: 16px;
+}
+
+.story-time {
+  margin-top: 2px;
+  font-size: 13px;
+  opacity: 0.5;
+}
+
+.side-recommand {
+  display: flex;
+  justify-content: space-between;
+  button {
+    margin-right: 10px;
+    border: none;
+    outline: none;
+    box-shadow: none;
+    background-color: transparent;
+    color: rgb(80, 149, 228);
+  }
+  button:hover {
+    cursor: pointer;
+  }
```

cursor 속성은 마우스를 올렸을 때 적용되는 속성이기 때문에 hover에 넣어두지 않아도 됨

---

또한 hover를 사용하고자 하는 경우 다음과 같이 부모 선택자(&)를 이용할 수 있음
```suggestion
  button {
    margin-right: 10px;
    border: none;
    outline: none;
    box-shadow: none;
    background-color: transparent;
    color: rgb(80, 149, 228);
      
    &:hover {
      ...
    }
  }
```